liquibase-demo-data.xml

```diff
@@ -21,5 +21,13 @@
 			<column name="MD5SUM" type="varchar(35)" />
 		</ext:modifyColumn>
 	</changeSet>
-	   
+	
+	<changeSet id="drop-tribe-foreign-key-TRUNK-4492" author="dkayiwa">
```

Comment:
@gracebish  is the author dkayiwa

Reply:
Sure

Reply:
@slubwama Did you see my last comment? It was based on https://github.com/openmrs/openmrs-core/commit/010be735a79865ed954a95400addf76ab8dfdb70#diff-7efe1c74929005aa8904f543f149c282